### DISCUSSION TITLE
Nix flake for Holesail

### DISCUSSION THREAD

**Discussion by gudnuf** [holesail/holesail#24] [Show and tell]
Nix allows you to easily make reproducible builds, and I want to deploy servers with holesail enabled. 

You can now run holesail without installing node! You just need the [nix package manager](https://nixos.org/).

Then you can use the [nix flake I made](https://github.com/gudnuf/holesail-nix/tree/main).

I am currently pulling in holesail from my fork for two reasons:
1. nix needs specific versions, and the release for holesail on github is out of date. If I were to point at the main branch of holesail, then every time there are updates, the flake would break. I solved this by pinning the flake to a branch.
2. `holesail-manager list` does not work without this fix: https://github.com/holesail/holesail/pull/23

**Comment by supersuryaansh** [comment]
🔥
